tests/pages/components/comment-item.js

```diff
@@ -39,5 +39,8 @@ export default {
     itemScope: '.error'
   }),
 
-  username: text('.username')
+  username: {
+    scope: '[data-test-username]',
+    text: text()
```

Comment:
I think it may be that you'll have a `text` property on `username` even without explicitly defining it. 

I'm reasonably sure all page elements (objects with `scope`) have a couple of these by default. Could you check and remove them if that's the case?